velox/dwio/common/Options.h

```diff
@@ -72,7 +72,34 @@ enum class SerDeSeparator {
 
 class SerDeOptions {
  public:
+  /// The following members control how data is separated TEXT file:
+  ///
+  /// - 'separators': An array of separator characters used to delimit columns
+  ///   and nested data.
+  ///     - 'separators[0]' separates top-level columns.
+  ///     - 'separators[depth_]' separates nested ComplexType columns.
+  /// - 'newLine': The character used to separate rows in the file.
+  ///
+  /// Suppose we have a schema: ROW(MAP(VARCHAR(), ARRAY(BIGINT())), BOOLEAN())
+  /// With the following configuration:
+  ///   - separators = [',', '@', ':', '#]
+  ///   - newLine = '\n'
+  ///   - nullString = "NULL"
+  ///
+  /// separators[0] is column-level separation (separates MAP and BOOLEAN)
+  /// separators[1] is pair-level separation (separates keys in MAP)
+  /// separators[2] is key-value separation (separates key and value)
+  /// separators[3] is key-complex-type separation (separates values within the
+  /// MAP value)
+  ///
+  /// A sample text file with 2 rows of data would look like this:
```

Comment:
For example: 2 rows of data 
row1: {{key1, [10, 20, 30]}, {key2, [40, 50, 60]}}, true
row2: {{key3, [100, 2, 30]}, {key4, [80, 40, 45]}}, true
 in TEXT format would look like this: